src/WooCommerce.php

```diff
@@ -552,11 +563,20 @@ public static function woocommerce_display_item_meta($html, $item, $args) {
     }
 
     foreach ($product_data_set as $productData) {
-      $strings[] = '<strong class="wc-item-meta-label">' . $productData['name'] . ':</strong> ' . $productData['value'];
+      $string = '';
+      if (!empty($productData['name'])) {
+        $string .= '<strong class="wc-item-meta-label">' . $productData['name'] . ':</strong> ';
+      }
+      if (!empty($productData['value'])) {
```

Comment:
What if the meta data value is zero? (`"0"`)

```suggestion
      if (isset($productData['value']) && strlen(trim($productData['value']))) {
```

Reply:
👍